index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>website concert</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d6d5f8b156.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+ <section class="homepage">
+  <nav class="mobile-navbar">
+      <div class="nav-logo">
+          <div id="hamburger">
+              <i class="fa fa-bars"></i>
+          </div>
+          <div class="logo">
+            <a href="index.html"></a>
+            <img class="img-logo" src="./images/riana.png" alt="Logo">
+          </div>
+      </div>
+      <div class="mobile-menu hide" id="mobile-menu-id">
+          <button id="cross">
+            <img src="./images/Icon.png" alt="">
+          </button>
+          <nav class="menu-list">
+              <ul>
+                  <li><a href="index.html">Home</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#services-id">program</a></li>
+                  <li><a href="#team-id">sponsor</a></li>
+                  <li><a href="#team-id">news</a></li>
+              </ul>
+          </nav>
+      </div>
+  </nav>
+  <div class="nav-top">
+      <div class="desktop-nav-list" id="nav-top-list">
+          <ul>
+              <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href=""><i class="fab fa-twitter"></i></a></li>
+              <li><a href="">English</a></li>
+              <li><a href=""> My Page</a></li>
+          </ul>
+      </div>
+  </div>
+  <div class="desktop-navbar">
+      <div class="logo">
+          <img src="./images/riana.png" alt="logo-image">
+      </div>
+      <div class="desktop-nav-list">
+          <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#team-id">program</a></li>
+              <li><a href="#team-id"> Sponsor</a></li>
+              <li><a href="#team-id"> News</a></li>
+          </ul>
+      </div>
+  </div>
+  </nav>
+  <div class="welcome">
+    <h3 class="text">"Hello how are you!"</h3>
+    <h1 class="text">Concert King's Jimin</h1>
+  </div>
+  <div class="welcome-description">
+    <p class="welcome-text">
+       Stage King' Jimin wows fans and netizens with his exhilarating dance,
+       vocals and visuals after BTS's Busan concert
+    </p>
+  </div>
```

Comment:
- [ ]  Please try to re-position here:
- [ ] Your design
- 
![Capture](https://user-images.githubusercontent.com/63972670/198000169-a26c9a6d-7341-4971-a845-ff7106877e0c.PNG)
- Original Design
![Capture](https://user-images.githubusercontent.com/63972670/198000346-5746dcc9-8369-460a-b199-b4ff224a92b1.PNG)